### ISSUE TITLE
sql/schema: support DDLs inside stored procedures and UDFs

### ISSUE THREAD

**Issue by dikshant** [cockroachdb/cockroach#126012]
We don't currently support DDLs inside stored procedures. We could perhaps support a limited set of DDLs such as create tables within a stored procedures. This would unblock a lot of use cases where temp tables are used inside stored procedures.

Jira issue: CRDB-39728

Jira issue: CRDB-39728

**Comment by mgartner** [comment]
Closing as a duplicate of #110080.
